wger/exercises/tests/test_exercise_info.py

```diff
@@ -1,11 +1,71 @@
 from wger.core.tests.api_base_test import (ApiGetTestCase, ApiBaseTestCase)
 from wger.core.tests.base_testcase import WorkoutManagerTestCase
 from wger.exercises.models import Exercise
+from rest_framework.test import APITestCase
+from rest_framework import status
 
 
-class TestExerciseInfo(ApiBaseTestCase, ApiGetTestCase, WorkoutManagerTestCase):
-
+class TestExerciseInfo(
```

Comment:
Could you consider adding docstrings to explain what the test does?

Reply:
I have added them. I believe now all is well.